lib/src/widgets/utils/ds_header.widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class DSHeader extends StatelessWidget implements PreferredSizeWidget {
+  final String title;
+  final String? subtitle;
+  final String? customerName;
+  final List<Widget>? actions;
+  final Widget? leading;
+
+  const DSHeader({
+    Key? key,
+    required this.title,
+    this.subtitle,
+    this.customerName,
+    this.actions,
+    this.leading,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      elevation: 1,
+      backgroundColor: DSColors.neutralLightSnow,
```

Comment:
In conversation with the UX, it was informed that the bar would have an opacity of 70% in the image and video widgets. Would it be useful to consider this for reuse?

Reply:
If it is not acceptable to open the color change to the user, perhaps implement it by including a flag, or create and export another widget inheriting from it, and containing this feature.